stock_picking_info_lot/views/stock_move_line_views.xml

```diff
@@ -65,7 +64,6 @@
                     name="lot_info"
                     attrs="{
                        'readonly': [('lot_info_usage', '=', 'no')],
-                       'required': [('lot_info_usage', '=', 'required')],
```

Comment:
@dreispt 
Maybe you can set a field that compute the 'required' condition of lot_info_usage for form, so user can see if the field is required when he filled in a quantity ?